modules/navier_stokes/src/fvkernels/PCNSFVMomentumPressureRZ.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PCNSFVMomentumPressureRZ.h"
+
+#include "NS.h"
+
+registerMooseObject("NavierStokesApp", PCNSFVMomentumPressureRZ);
+
+InputParameters
+PCNSFVMomentumPressureRZ::validParams()
+{
+  InputParameters params = FVElementalKernel::validParams();
+  params.addClassDescription(
+      "Adds the porous $-p/r$ term into the radial component of the Navier-Stokes "
+      "momentum equation for the problems in the RZ coordinate system.");
+  return params;
+}
+
+PCNSFVMomentumPressureRZ::PCNSFVMomentumPressureRZ(const InputParameters & params)
+  : FVElementalKernel(params),
+    _p(getADMaterialProperty<Real>(NS::pressure)),
```

Comment:
This is a material property though and in all the `VarMaterials` the property is declared as `NS::pressure`. In this PR I also added `prop_getter_suffix` as a parameter to `MaterialPropertyInterface` and `declare_suffix` to `MaterialBase` to allow more flexibility in property naming when there are hard-coded property names. 

When a material declares properties, do we want to add `MaterialPropertyName` parameters for every declared property (with defaults perhaps)? This could become overload. At least when using PCNSFV a user will always want to be using one of the `VarMaterials` and I think declaring and retrieving properties based on the NS namespace is a decent design.

I suppose that this would be more of an issue if we rename this to `PNSFVMomentumRZ` implying this can be used in incompressible cases where we will not be using the `VarMaterials`.

Reply:
I think in the future we can consider adding more flexibility here if needed